File: SecondYear/jeezzzz_AjeeshRawal_2226cseai1021_2/Week_1/pattern6.java
class Solution {

    void printTriangle(int n) {
        int space=n-1,i,j;
       for (j = 1; j<= n; j++)
         {
           for (i = 1; i<= space; i++)
            {
              System.out.print(" ");
            }
           space--;
           for (i = 1; i <= 2 * j - 1; i++)
            {
              System.out.print("*");
            }
           System.out.println("");
          }
    }
}

File: SecondYear/Akash-Tiwari22_AkashTiwari_2226cs1058_2/Week_1/pattern6.cpp
class Solution {
  public:
    void printTriangle(int n) {
        // code here
        for(int i=1;i<=n;i++)
        {
            for(int space=1;space<=n-i;space++)
            {
                cout<<" ";
            }
           
            for(int j=1;j<=(2*i)-1;j++)
            {
                cout<<"*";
            }
            cout<<endl;
        }
    }
};